vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/Application.java

```diff
@@ -22,8 +22,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.support.SpringBootServletInitializer;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
 @SpringBootApplication
+@EnableAutoConfiguration(exclude={DataSourceAutoConfiguration.class})
```

Comment:
If no datasource is needed, then we should simply remove the connection parameters from the properties files, and the auto-configuration shouldn't be needed because there is nothing to configure.

Reply:
The WAR file generates an error on deployment when executed without disabling the datasource autoconfiguration, as spring-boot expects a datasource by default. [Here](https://stackoverflow.com/questions/32074631/spring-boot-application-without-a-datasource) is a supporting link for the same.

Reply:
You're right, I read it the other way around, as if the data source autoconfiguration was **in**cluded. I still suggest removing the properties that do not apply from the corresponding files.